src/components/Nav/Nav.module.css

```diff
@@ -66,6 +66,7 @@
   background: transparent;
   border: 0;
   font: var(--cpd-font-body-md-regular);
+  font-weight: 500;
```

Comment:
is there no var for this? Surely CPD should have the right weight set for `Web/font/body/md/medium`

https://github.com/element-hq/compound-design-tokens/blob/240fb7dba469eccc963472033b489ea9be9b1fbf/assets/web/css/cpd-common-base.css#L53
https://github.com/element-hq/compound-design-tokens/blob/240fb7dba469eccc963472033b489ea9be9b1fbf/assets/web/css/cpd-common-base.css#L23